docs/UserGuide.md

```diff
@@ -14,9 +17,9 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 1. Ensure you have Java `17` or above installed in your Computer.
 
-1. Download the latest `.jar` file from [here](https://github.com/se-edu/addressbook-level3/releases).
+1. Download the latest `.jar` file from [here](https://github.com/AY2425S1-CS2103T-F13-1/tp/releases).
```

Comment:
Everything shows up as step 1 on Github. Is this intentional or a display error?

Reply:
Good Catch. Made a Push to change the Indices in Quick Start